### ISSUE TITLE
Search Page Title i18n?

### ISSUE THREAD

**Issue by UM-Li** [probberechts/hexo-theme-cactus#270]
The official method for enabling the [hexo-generate-search](https://www.npmjs.com/package/hexo-generator-search) plugin is to create a separate page.  The title is given in the front-matter, and its language does not change as other menu entries such as "about" or "writing" do. Is there a recommended way to add i18n for the search page title?

**Comment by probberechts** [comment]
I am a bit confused, since the title of the about page should not be translated automatically. Are you actually talking about the title of the page (the H1) or the link in the navigation menu? Also, is your site multi-language or single-language?

**Comment by UM-Li** [comment]
Ahh... sorry I got really confused yesterday and was absent-minded when writing the issue.  No, the title of the "about" page is not translated either. I am talking about the `<title>`, the one that appears in browser tabs. As far as this title is concerned the "about" page and the "search" page behave the same -- in contrast to the "writing" page.

But I just realized I have not really tested Cactus' i18n functionality by setting multiple languages, because Hexo signaled errors when I did so.  I will open another issue about that error first.
